terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/token/transaction/TransactionTokenInterceptor.java

```diff
@@ -388,11 +393,12 @@ void removeToken(TransactionToken receivedToken) {
     }
 
     /**
-     * [todo] write javadoc
-     * @param request
-     * @param tokenInfo
-     * @param receivedToken
-     * @param tokenStore
+     * To set the receivedToken to the attributes named <code>TransactionTokenInterceptor.NEXT_TOKEN</code> of request .
```

Comment:
- [ ] To set -> Set
- [ ] request . -> request.
- [ ] "without updates." at the last of the sentence